pyproject.toml

```
@@ -1,3 +1,69 @@
+[project]
+name = "khal"
+dynamic = ["version"]
+description = "A standards based terminal calendar"
```

Usually including `A ...` in package descriptions is discouraged. In the same way that we wouldn't use `This is a ...`

```suggestion
description = "Standards based terminal calendar"
```